src/test/java/com/exasol/projectkeeper/ProjectKeeperIT.java

```diff
@@ -58,7 +58,7 @@ void testVerify() throws IOException, InterruptedException {
                 () -> assertThat(result.getExitCode(), not(is(0))),
                 () -> assertThat(result.getStdout(), containsString("E-PK-6")),
                 () -> assertThat(result.getStdout(),
-                        containsString("E-PK-17: Missing required: .settings/org.eclipse.jdt.core.prefs")),
+                        containsString("E-PK-17: Missing required: '.settings/org.eclipse.jdt.core.prefs'")),
```

Comment:
I think it's enough to check an error code if there are no variables in the message. It will allow you to change the message slightly without breaking the tests

Reply:
well yes. In that case, I would like to check that this particular file had errors.

Reply:
Maybe then two assertions? 1. containsString("E-PK-17") 2. E-PK-17(".settings/org.eclipse.jdt.core.prefs") 
But if you prefer how it looks now, it's fine